src/game/sound_samples.c

```diff
@@ -33,28 +111,49 @@ Sound_samples *create_sound_samples(const char *sample_files[],
     sound_samples->lt = lt;
 
     sound_samples->samples_count = sample_files_count;
+    if (init_buffer_and_device(sound_samples, sample_files) < 0) {
+        log_fail("init_buffer_and_device failed\n");
+        RETURN_LT(lt, NULL);
+    }
+
     sound_samples->paused = 0;
+    SDL_PauseAudioDevice(sound_samples->dev, 0);
 
     return sound_samples;
 }
 
 void destroy_sound_samples(Sound_samples *sound_samples)
 {
+    // TODO: Use a seperate callback function for processing audio
```

Comment:
Duplicate TODO. Unfortunately, snitch does not support several TODOs per single issue.